### PR TITLE
chore: remove next components [BREAKING]

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/6717338787

### CODE REVIEW

File: packages/core/src/components/EditableInput/EditableInput.module.scss
Comment (talkor) [inline packages/core/src/components/EditableInput/EditableInput.module.scss:1]: Was used internally for the old EditableHeading, it was an unstyled input, but for some reason was exported so I added it to the changelog anyway

File: packages/core/.storybook/preview.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React from "react";
 import * as VibeComponents from "../src/components";
-import * as VibeComponentsNext from "../src/next";
```
Comment (shaharzil) [inline packages/core/.storybook/preview.tsx:3]: Let's consider not removing it completely but leaving it as is and just removing the components imports inside

Comment (talkor) [reply packages/core/.storybook/preview.tsx:3]: Brought it back

File: packages/core/package.json [hunk 2]
```diff
@@ -29,9 +29,9 @@
     },
     "./storybookComponents": "./dist/storybook/index.js",
     "./next": {
-      "types": "./dist/esm/next/next.d.ts",
-      "import": "./dist/esm/src/next/next.js",
-      "default": "./dist/esm/src/next/next.js"
+      "types": "./dist/esm/components/next/next.d.ts",
```
Comment (talkor) [inline packages/core/package.json:32]: There were two files which one of them was redundant, so now I pointed it to the other file (makes more sense)

File: packages/core/rollup.config.js [hunk 3]
```diff
@@ -67,8 +67,7 @@ export default {
     index: path.join(SRC_PATH, "index.js"),
     icons: path.join(SRC_PATH, "components/Icon/Icons/index.ts"),
     interactionsTests: path.join(SRC_PATH, "tests/interactions-utils.ts"),
-    testIds: path.join(SRC_PATH, "tests/test-ids-utils.ts"),
-    next: path.join(SRC_PATH, "next/next.ts")
```
Comment (shaharzil) [inline packages/core/rollup.config.js:71]: dont we also need to add this line back?

File: packages/core/src/storybook/decorators/withLiveEdit/components/LiveContent/LiveContent.tsx [hunk 4]
```diff
@@ -6,9 +6,8 @@ import { LiveContentProps } from "./LiveContent.types";
 import styles from "./LiveContent.module.scss";
 import * as VibeComponents from "../../../../../components";
 import * as VibeIcons from "../../../../../components/Icon/Icons";
-import * as VibeComponentsNext from "../../../../../next";
```
Comment (shaharzil) [inline packages/core/src/storybook/decorators/withLiveEdit/components/LiveContent/LiveContent.tsx:9]: dont we want to add next back?

File: packages/core/.storybook/preview.tsx (hunk 1 above)
Comment (talkor) [reply packages/core/.storybook/preview.tsx:3]: Moved to line 4

File: packages/core/rollup.config.js (hunk 3 above)
Comment (talkor) [reply packages/core/rollup.config.js:71]: Good catch, fixing

File: packages/core/src/storybook/decorators/withLiveEdit/components/LiveContent/LiveContent.tsx (hunk 4 above)
Comment (talkor) [reply packages/core/src/storybook/decorators/withLiveEdit/components/LiveContent/LiveContent.tsx:9]: good catch, fixing

**Review by shaharzil — APPROVED** [review]

**Pull request merged**